packages/v4/src/content/design-guidelines/charts/about/about.md

```diff
@@ -0,0 +1,61 @@
+---
+id: About
+section: charts
+---
+
+_Charts are only available in React_
+
+Use charts to visualize data in an application. The type of chart you use will depend on your use case and the type of data you need to display.
+
+## Elements
+<img src="./img/axis-labels.png" alt="Chart with axis labels" width="713"/> 
+
+1. **Title:** Titles should be left-aligned and appear above your chart.
+2. **Axis labels and scale values:**  We recommend using tick marks to clearly mark scale values on the y-axis. The text for axis labels and scale values should be 12px font in standard text color. All scale values should be equally distributed across the axis and the axis label should fall outside the scale values, centered with the axis line.
+3. **Legends:** We recommend using legends when charts include more than one variable and therefore use more than one color. When datasets are overlapping, an [interactive legend](#interactive-chart-legends) can be used. If space is not available to display names of horizontal and vertical axes, a legend must be added instead. The legend should list the colors that each variable obtains and the name of the variable. The text on the legend should be 14px size and standard text color. They can either be placed left-aligned to the right of the chart or centered below the chart.
+4. **Chart tooltip:** A tooltip will appear on hover over certain elements of a chart, like bars in a bar chart or segments in a donut chart. [Chart tooltips](/charts/tooltip-chart) should display the specific values related to the element the user is hovering over. 
+
+## Chart types
+- [Area charts](/charts/area-chart/design-guidelines): use to show trending over a continuous scale (usually time)
+- [Bar charts](/charts/bar-chart/design-guidelines): great way to show and compare categorical data
+  - [Stacked bar charts](/charts/stacked-chart/design-guidelines)
+- [Bullet charts](/charts/bullet-chart): use as an extension to the bar chart to measure data with thresholds and static ranges
```

Comment:
Shouldn't this be going to the Bullet chart design guidelines page for consistency with other links?

Reply:
Oh yes, good catch!